health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/index.js

```
@@ -26,8 +30,8 @@ function DeliverySetup({ onSelect, config, formData, control, tabCount = 2, subT
     "HCM-PROJECT-TYPES",
     [{ name: "projectTypes" }],
     {
-      staleTime: 0,
-      cacheTime: 0,
+      staleTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)